src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/WebJobs.Extensions.ServiceBus.csproj

```diff
@@ -28,6 +29,7 @@
     <StyleCopTreatErrorsAsWarnings>false</StyleCopTreatErrorsAsWarnings>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <WarningsAsErrors />
+    <NoWarn>1701;1702;NU5125;NU5048</NoWarn>
```

Comment:
why the warnings were excluded?

Reply:
This is to suppress the warning: https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5125 until we change the way we link to the license.  I can remove the suppression if we don't want it in the repo.

Reply:
Reverted